cmd/main.go

```diff
@@ -3,32 +3,38 @@ package main
 import (
 	"bufio"
 	"bytes"
-	"fmt"
-	"io"
 	"log"
 	"os"
+	"os/user"
 	"spy"
+
+	"github.com/fatih/color"
 )
 
 func main() {
 
 	scanner := bufio.NewScanner(os.Stdin)
 	f, err := os.Create("/tmp/log")
+	if err != nil {
+		log.Fatal(err)
+	}
 
+	user, err := user.Current()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	host, err := os.Hostname()
```

Comment:
Customising the prompt is a lovely idea! We might even support something like the same language that users will be familiar with for setting their bash prompt, mightn't we? https://phoenixnap.com/kb/change-bash-prompt-linux